src/adapter/src/coord/read_policy.rs

```diff
@@ -37,21 +38,24 @@ use timely::progress::Antichain;
 
 use crate::coord::id_bundle::CollectionIdBundle;
 use crate::coord::timeline::{TimelineContext, TimelineState};
-use crate::coord::Coordinator;
 use crate::session::Session;
 use crate::util::ResultExt;
 
 /// Relevant information for acquiring or releasing a bundle of read holds.
-#[derive(Debug, Serialize)]
+#[derive(Serialize)]
 pub struct ReadHolds<T> {
     pub holds: HashMap<Antichain<T>, CollectionIdBundle>,
+    // We only use Serialize for debugging.
```

Comment:
Should we just replace the `Serialize` usages with `Debug`?

Reply:
I think a bunch of `Serialize` bounds were added for the "dump coordinator state" work that @ParkMyCar did. That uses `Serialize` to dump a json view of coordinator state, or something like it.

Reply:
I think the `Serialize` impl for `ReadHolds` is unneeded since https://github.com/MaterializeInc/materialize/pull/26237

Reply:
+1 you should be able to get rid of the `Serialize` bound